### ISSUE TITLE
Automatic Generation of MO Files between 3.4dev and 4.3 different

### ISSUE THREAD

**Issue by TheWitness** [WeblateOrg/weblate#5036]
**Describe the issue**

Our current setup is as follows:

src/locales/po/*.[po|pot]
src/locales/LC_MESSAGES/*.mo

With the 3.4dev version of Weblate, we were able to add the "Automatically Generate MO files" add-on module and set the path to:

../LC_MESSAGES/

However, in the current release, that does not seem to work.  What am I doing wrong?

**I already tried**

Using the same setting as 3.4dev version.

**Comment by nijel** [comment]
> Using the same setting as 3.4dev version.

I don't think the addon had any settings in the 3.4 or 3.5 release (hard to guess what you were closer to), so I'm not sure about which settings are you talking.

Custom location for generated MO files is supported since a6a075c1e97 which was part of the 3.7 release and I don't think the feature has been changed since then - it uses [template markup](https://docs.weblate.org/en/latest/admin/projects.html#markup).

**Comment by TheWitness** [comment]
Okay, my guess is that one of our guys likely back ported it.  We had to do a complete re-install and reload of the entirety of the tool to upgrade as the strait upgrade did now work, but when I added that path which is on the legacy site to the add-on, it did not work.

This is what I tried to use and it did not work of course:

![image](https://user-images.githubusercontent.com/1439914/102088172-8a7bac80-3de8-11eb-9340-6c516977b410.png)

**Comment by TheWitness** [comment]
This is the result.

![image](https://user-images.githubusercontent.com/1439914/102088295-bac34b00-3de8-11eb-9727-1fdc430da7e0.png)

**Comment by TheWitness** [comment]
Just found this though.  How did that happen?

![image](https://user-images.githubusercontent.com/1439914/102088459-f231f780-3de8-11eb-96fa-730619874c62.png)

**Comment by nijel** [comment]
It might be that the validation was not that strict in previous releases, using .. is certainly not desired here.

**Comment by TheWitness** [comment]
Well, I think the absolute path inside the repo was accepted.  I've applied it to all components.  Still having ssh issues that I need to track down, but getting closer to having it working again.

We keep having issues with the Merge since we are making lot's of source changes, and it seems that we can not ignore them.  So, when weblate tries to Merge, it get's a number of conflicts.  I saw the documentation on 'sub components'.  However, that is new to me.
